docs/course.md

```diff
@@ -124,6 +124,31 @@ The assessment set order in `infoCourse.json` is the order in which the assessme
     ],
 ```
 
+## Units
+
+Each assessment in the course belongs to a *unit* defined in `infoCourse.json`. Units should be thought of as a unit of instruction, typically with a consistent number of assessments per unit, such as 3 homeworks, 2 labs, and 1 exam. Properties for assessment units are as follows.
+
+Property | Description
+--- | ---
+`name` | One word to identify the unit, lowercase.
+`heading` | Longer title for the unit, displayed to users.
+
+Units are optional and do not affect any behavior by default. Setting the `infoCourse.json` property `"groupBy"` to `"unit"` (over the default `"set"`) will cause assessments to be grouped together by unit on the student assessments overview page. Because units are defined in an array, the order that they are defined in will be the order that they display in, top-down, to students. This can let students view their list of assessments in a chronological order, rather than simply grouped by set.
```

Comment:
```suggestion
Units are optional and do not affect any behavior by default. Setting the `infoCourse.json` property `"groupBy"` to `"unit"` (instead of the default `"set"`) will cause assessments to be grouped together by unit on the student assessments overview page. The order that they are defined in will be the order that they display in, top-down, to students. This can let students view their list of assessments in a chronological order, rather than simply grouped by set.
```